dvc/remote/gdrive.py

```diff
@@ -54,6 +56,18 @@ def should_retry(exc):
     )(func)
 
 
+def location(exc):
```

Comment:
`def _extract_error_details`

I would also extract `location` and `reason` and check both of them?

Reply:
actually let's just do def `_location` - I think it's fine ... just put a comment what it does with a link to error strucutre